metadata-io/src/main/java/com/linkedin/metadata/entity/EntityService.java

```diff
@@ -170,6 +170,9 @@ public static class IngestProposalResult {
 
   private static final int URN_NUM_BYTES_LIMIT = 512;
 
+  // TODO(iprentic): Move this to a common utils location once used in other places
+  private static final String DELIMITER_SEPARATOR = "␟";
```

Comment:
Might be less visually weird to have this as the character code rather than a blank looking String in case someone edits this without understanding it.